.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+pom.xml @andrewazores @ebaron
+src/main/docker/ @andrewazores @ebaron
```

Comment:
I think this can reference `@cryostatio/maintainers` rather than listing Elliott and I specifically, now that we are using org teams.